### PR TITLE
chore: Fix outdated stuff

### PR DESCRIPTION
**Pull request by Bavfalcon9**
Fixes versioning bugs

### CODE REVIEW

File: chrome.ts [hunk 1]
```diff
@@ -238,7 +238,7 @@ class ChromeImpl implements Chrome {
         return;
       }
 
-      const folder = entry.folder;
+      const folder = entry.folder?.split('/').slice(1).join();
```
Comment (uki00a) [inline chrome.ts:241]: Why is this change needed:question::thinking:

**Review by uki00a — CHANGES_REQUESTED** [review]
@Bavfalcon9 Thank you for submitting this PR! Looks good to me, but please make a few changes before merging this PR:
* Please run `deno fmt`.
* I leaved a comment. Please check it out!

File: chrome.ts (hunk 1 above)
Comment (Bavfalcon9) [reply chrome.ts]: This is needed for module resolution. (At least form my tests)

Comment (uki00a) [reply chrome.ts]: Oh, I see. :slightly_smiling_face:  Maybe it's an OS specific bug. :cry: (This module is currently being tested only on Linux) What OS are you using?

Comment (uki00a) [reply chrome.ts]: In any case, I think this bug should be fixed in a separate PR. Therefore, please let me revert for this change :wink: (I'll open an issue later)

File: chrome.ts [hunk 2]
```diff
@@ -238,7 +238,7 @@ class ChromeImpl implements Chrome {
         return;
       }
 
-      const folder = entry.folder;
+      const folder = entry.folder?.split("/").slice(1).join();
```
Comment (uki00a) [inline chrome.ts:241]: In any case, I think this bug should be fixed in a separate PR. :thinking:  So, could you please revert this change? (I'll open an issue later about this bug)
```suggestion
      const folder;
```

**Review by uki00a — CHANGES_REQUESTED** [review]

File: chrome.ts (hunk 1 above)
Comment (Bavfalcon9) [reply chrome.ts]: Windows

File: chrome.ts [hunk 3]
```diff
@@ -238,7 +238,7 @@ class ChromeImpl implements Chrome {
         return;
       }
 
-      const folder = entry.folder;
+      const folder;
```
Comment (uki00a) [inline chrome.ts:241]: Sorry, there was a mistake in the code I posted in the previous comment https://github.com/uki00a/carol/pull/49#discussion_r542377751.
```suggestion
      const folder = entry.folder;
```

**Review by uki00a — CHANGES_REQUESTED** [review]

**Review by uki00a — APPROVED** [review]
LGTM! thank you for the contribution!
